Assets/Scripts/Combat/Abilities/ThunderStorm.cs

```diff
@@ -248,4 +261,15 @@ private bool WithinPerfectStrikeWindow(float timerValue)
         return timerValue >= ((TimeWindowForStrikes - PerfectStrikeTimeWindow) / 2.0f) &&
                     timerValue <= ((TimeWindowForStrikes + PerfectStrikeTimeWindow) / 2.0f);
     }
+
```

Comment:
As I'm explaining to you now, the timing of the thunderstrike will need to be moved